redbot/core/cog_manager.py

```diff
@@ -7,7 +7,7 @@
 from typing import TYPE_CHECKING, Union, List, Optional
```

Comment:
```suggestion
from typing import Union, List, Optional
```
`TYPE_CHECKING` is no longer used inside the cog_manager file, we can remove its import.